ci/stage-test-sgx.jenkinsfile

```diff
@@ -162,11 +163,24 @@ stage('test-sgx') {
 
     try {    
         timeout(time: 5, unit: 'MINUTES') {
-            sh '''
+        if(env.no_cpu.toInteger() > 16)
+            {
+                sh '''
                 cd Examples/blender
+                sed -i 's/sgx.enclave_size = "2048M"/sgx.enclave_size = "4G"/g' blender.manifest.template
+                sed -i 's/sgx.thread_num = 64/sgx.thread_num = 256/g' blender.manifest.template
                 make ${MAKEOPTS} all
```

Comment:
make all and make SGX=1 is duplicate. Move them out of the if else block. Those commands need to run irrespective. Even cd Examples/blender is common and should our before the if else block